property/index.js

```diff
@@ -166,3 +156,68 @@ module.exports = yeoman.generators.Base.extend({
 
   saveProject: actions.saveProject
 });
+
+function coerceDefault(def, value) {
+  var isSupported = typeChoices.indexOf(def.type) !== -1;
```

Comment:
It is a valid use case to provide a custom type, e.g. a Model name. I think a better variable name is `isBuiltin`.